### ISSUE TITLE
Tinfoil Hat emoji

### ISSUE THREAD

**Issue by Crissov** [Crissov/unicode-proposals#188]
A Tinfoil Hat or a Person Wearing a Tinfoil Hat emoji would be used in the context of conspiracy theories and to describe someone as lunatic.

**Comment by AWqxKAWERbXo** [comment]
Is there any progress? Like an official proposal through https://unicode.org/emoji/proposals.html#emoji_proposals_form

**Comment by Crissov** [comment]
No, apparently not. <https://unicode.org/emoji/emoji-requests.html>

**Comment by mitra42** [comment]
At least its in slack. :tinfoil_hat:

**Comment by AWqxKAWERbXo** [comment]
2020 showed us that this Emoji needs high priority.

I'll try to suggest the proposal, never did that before and English is not my mother tounge.

**Comment by Crissov** [comment]
No formal proposal for _Tin-foil Hat_ or _Face with Tinfoil Hat_ has been recorded by the ESC: https://unicode.org/emoji/emoji-requests.html

You could ask the kind and experienced folks at [Emojination](https://Emojination.org) for assistance, e.g. when it comes to sample graphics.
